projects/libwebrtc/.gitignore

```diff
@@ -0,0 +1,4 @@
+.config.sh
```

Comment:
Same, can we merge all `.gitignore`?

Reply:
It isn't safe to assume that files matching these filenames should be excluded everywhere across the project. Do you prefer root-level entries like:
```
tools/build/libwebrtc/.config.sh
```
?

Reply:
Yes, I think it's easier to understand when there's a single root `.gitignore`. And marginally faster for git to read a single file.

Reply:
.gitignores merged into the root